build.gradle

```diff
@@ -50,8 +44,8 @@ ext.deps = [
     "avro": "org.apache.avro:avro:1.8.2",
     "awaitility": "org.awaitility:awaitility:2.0.0",
     "commons_io": "commons-io:commons-io:2.4",
-    "guava": "com.google.guava:guava:16.0.1",
-    "guice": "com.google.inject:guice:4.1.0",
+    "guava": "com.google.guava:guava:11.0.2",
+    "guice": "com.google.inject:guice:3.0",
```

Comment:
Did using newer Guava and Guice versions cause compilation issues for you? Hadoop 2.9 is also on Guava 11.0.2 and Guice 3.0 but compiles fine using newer versions.

Reply:
Yes, the newer version don't compile against 2.7 :(